python/sglang/srt/server_args.py

```diff
@@ -273,6 +281,7 @@ def add_cli_args(parser: argparse.ArgumentParser):
                 "gptq_marlin",
                 "awq_marlin",
                 "bitsandbytes",
+                "compressed-tensors",
```

Comment:
Does this feature only comes with `compressed-tensors`?
Can we decouple a bit, and add torchao's INT4/INT8 support too?

Reply:
Can reuse a8w8 linear in vllm